### PR TITLE
Update directory structure and package names

### PR DESCRIPTION
**Pull request by chamil321**
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Ballerina.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ name = "mime"
 version = "@toml.version@"
 
 [[platform.java11.dependency]]
-path = "../mime-native/build/libs/mime-native-@project.version@.jar"
+path = "../native/build/libs/mime-native-@project.version@.jar"
 groupId = "ballerina"
 artifactId = "mime"
 version = "@project.version@"
```
Comment (ThisaruGuruge) [inline ballerina/Ballerina.toml:10]: We can remove these entries.

File: native/src/main/java/io/ballerina/stdlib/mime/nativeimpl/MimeEntityBody.java [hunk 2]
```diff
@@ -45,27 +45,11 @@
...
-import static org.ballerinalang.mime.util.MimeConstants.STREAM_ENTRY_RECORD;
-import static org.ballerinalang.mime.util.MimeConstants.TEXT_PLAIN;
-import static org.ballerinalang.mime.util.MimeUtil.getContentTypeWithParameters;
-import static org.ballerinalang.mime.util.MimeUtil.getMimePackage;
-import static org.ballerinalang.mime.util.MimeUtil.getNewMultipartDelimiter;
+import static io.ballerina.stdlib.mime.nativeimpl.MimeDataSourceBuilder.getErrorMsg;
+import static io.ballerina.stdlib.mime.util.HeaderUtil.isMultipart;
+import static io.ballerina.stdlib.mime.util.MimeUtil.getContentTypeWithParameters;
+import static io.ballerina.stdlib.mime.util.MimeUtil.getMimePackage;
+import static io.ballerina.stdlib.mime.util.MimeUtil.getNewMultipartDelimiter;
```
Comment (ThisaruGuruge) [inline native/src/main/java/io/ballerina/stdlib/mime/nativeimpl/MimeEntityBody.java:52]: Is this intentional?

File: native/src/main/java/io/ballerina/stdlib/mime/util/HeaderUtil.java [hunk 3]
```diff
@@ -34,14 +34,6 @@
 import javax.activation.MimeType;
 import javax.activation.MimeTypeParseException;
 
-import static org.ballerinalang.mime.util.MimeConstants.ASSIGNMENT;
-import static org.ballerinalang.mime.util.MimeConstants.BOUNDARY;
-import static org.ballerinalang.mime.util.MimeConstants.FIRST_ELEMENT;
-import static org.ballerinalang.mime.util.MimeConstants.INVALID_HEADER_PARAM_ERROR;
-import static org.ballerinalang.mime.util.MimeConstants.INVALID_HEADER_VALUE_ERROR;
-import static org.ballerinalang.mime.util.MimeConstants.MULTIPART_AS_PRIMARY_TYPE;
-import static org.ballerinalang.mime.util.MimeConstants.SEMICOLON;
```
Comment (ThisaruGuruge) [inline native/src/main/java/io/ballerina/stdlib/mime/util/HeaderUtil.java:43]: Is this intentional?

**Review by ThisaruGuruge — COMMENTED** [review]
Dependencies in the build.gradle should be updated with the new artifact group name.

File: ballerina/build.gradle [hunk 4]
```diff
@@ -59,7 +59,7 @@ ballerina {
     packageOrganization = packageOrg
     module = packageName
     langVersion = ballerinaLangVersion
-    testCoverageParam = "--code-coverage --coverage-format=xml --includes=org.ballerinalang.mime.*:ballerina.mime.*"
+    testCoverageParam = "--code-coverage --coverage-format=xml --includes=io.ballerina.stdlib.mime.*:ballerina.mime.*"
```
[the remark this answers is not in the document]
Comment (chamil321) [reply ballerina/build.gradle:62]: you meant to remove whole line?

[the remark this answers is not in the document]
Comment (niveathika) [reply ballerina/build.gradle]: yes

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**